GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -383,30 +409,25 @@ extension Glia {
             switch event {
             case .screenShareOffer(answer: let answer):
                 self?.environment.coreSdk.requestEngagedOperator { operators, _ in
-                    self?.callVisualizer.offerScreenShare(
-                        from: operators ?? [],
-                        configuration: Theme().alertConfiguration.screenShareOffer,
+                    self?.operatorRequestHandlerService.offerScreenShare(
+                        from: operators?.compactMap { $0.name }.joined() ?? "",
                         accepted: { [weak self] in
-                            self?.loggerPhase.logger.prefixed(Self.self).info("Screen sharing accepted by visitor")
-                            answer(true)
+                            self?.callVisualizer.observeScreenSharingHandlerState()
                         },
-                        declined: {
-                            self?.loggerPhase.logger.prefixed(Self.self).info("Screen sharing declined by visitor")
-                            answer(false)
-                        }
+                        declined: {},
```

Comment:
don't `declined` is not necessary? if so, would it make sense to add coment with reason why it is not applicable for this case, since for now it looks like someone forget to add implementation.

Reply:
Answer and decline callback changed its purpose a bit. Before either one was responsible for providing the answer to the coreSDK, and doing other things such as logging and calling a method. Now they are exclusively for calling a necessary method. What I can do is make accept and decline optional, and using them only where needed.